### DISCUSSION TITLE
Nextjs 13.5.4 Potential Issue

### DISCUSSION THREAD

**Discussion by kschmott** [thebuilder/react-intersection-observer#647] [General]
Hey all,

I am just curious if anyone else is encountering issues with the InView Component not working(onChange never fires) in Nextj 13.5.4 but working on 13.4.19? It could be a me problem, but thought I'd ask.

Thanks!

**Comment by thebuilder** [comment]
See https://github.com/thebuilder/react-intersection-observer/issues/649

Make sure you actually add `'use client'` to your component/page.
